compatibility-kit/javascript/Makefile

```diff
@@ -2,13 +2,13 @@ include default.mk
 
 FEATURE_FILES = $(sort $(wildcard features/**/*.feature features/**/*.md))
 NDJSON_FILES = $(patsubst features/%,features/%.ndjson,$(FEATURE_FILES))
+VALIDATED_NDJSON_FILES = $(patsubst features/%.ndjson,acceptance/%.ndjson,$(NDJSON_FILES))
 
 .DELETE_ON_ERROR:
 
-.tested: ndjson_files
-
-ndjson_files: $(NDJSON_FILES)
-.PHONY: ndjson_files
+.tested: $(NDJSON_FILES)
+# TODO: Uncomment next line and remove previous line
```

Comment:
@aurelien-reeves @mattwynne I have added a validation utility that validates all of the CCK messages against the schema. It's currently disabled, but if you comment out the next line you can see it fail.

To validate a specific file, run:

    make acceptance/minimal/minimal.feature.ndjson.valid

I think we should make further changes to the schemas to make these validation checks pass. Can you two have a look at that?

Reply:
It was an easy enough fix - done!